.travis.yml

```diff
@@ -54,9 +54,11 @@ script:
   - if [ -n "$DOCKERFILE" ]; then
       docker run "$DOCKERFILE"  /bin/sh -c "cmake -DOMPL_REGISTRATION=OFF .. && make -j `nproc` && ctest";
     else
-      mkdir -p build &&
-      cd build &&
-      cmake -G Ninja -DOMPL_REGISTRATION=OFF .. &&
-      cmake --build . &&
-      ctest;
+      cmake -B build -G Ninja -DOMPL_REGISTRATION=OFF &&
+      cmake --build build &&
+      cmake --build build &&
```

Comment:
Why the double build here?

Reply:
And the third build later.

Reply:
Double build is accidental, good catch. Third build is checking the CMake export is functional.

Reply:
Double build fix in 4d6a5ab1f3831323e1f55aca7f71b441986736f2